### DISCUSSION TITLE
Commit message: how to not show  split window below & how to set default filetype

### DISCUSSION THREAD

**Discussion by uwla** [NeogitOrg/neogit#1441] [Q&A] [answered]
When editing the commit message:

- It shows a split window below, `NeogitDiffView`. How to not show that?
- How to have the commit message filetype to `gitcommit` instead of`NeogitCommitMessage`?

**Comment by gldtn** [comment, marked as answer]
> * It shows a split window below, `NeogitDiffView`. How to not show that?

I believe you're looking for this: 

```lua
commit_editor = { show_staged_diff = false },
```

**Comment by CKolkey** [comment]
WRT: filetype - I think I'll cut a breaking change soon that changes this.
